frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts

```diff
@@ -372,6 +411,7 @@ export const heatmapsBrowserLogic = kea<heatmapsBrowserLogicType>([
         '/heatmaps': (_, searchParams) => {
             if (searchParams.pageURL && searchParams.pageURL !== values.browserUrl) {
                 actions.setBrowserUrl(searchParams.pageURL)
+                actions.setDisplayUrl(searchParams.pageURL)
```

Comment:
if we always want to set one when we set the other (which on a scan it looks like we do) then we could listen to setDisplayUrl and set the data url when it changes?